cache/remotecache/import.go

```diff
@@ -302,8 +302,9 @@ type image struct {
 	Rootfs struct {
 		DiffIDs []digest.Digest `json:"diff_ids"`
 	} `json:"rootfs"`
-	Cache   []byte `json:"moby.buildkit.cache.v0"`
-	History []struct {
+	Cache              []byte `json:"moby.buildkit.cache.v0"`
+	EarthlyInlineCache []byte `json:"earthly.inlinecache.v0"`
```

Comment:
This field is added to an image's "config" and can be viewed using something like `crane config`. The map of cache keys to registry layer digests is stored here.